src/server.c

```diff
@@ -416,7 +344,14 @@ main(int argc, char **argv) {
     lwsl_notice("tty configuration:\n");
     if (server->credential != NULL)
         lwsl_notice("  credential: %s\n", server->credential);
-    lwsl_notice("  start command: %s\n", server->command);
+    
+    lwsl_notice("  start command: \n");
+    for (c = 0; c < argc; c++) {
+        lwsl_notice(" %s\n", argv[c]);
```

Comment:
<What you d like is not what you need>

Do you rather
 * quote the command or the shell, lie `ttyd "[ $(expr `date +%s` % 2) == 0 ] && ls || date"` or `ttyd "/bin/bash -l"` to get a one line 
 * wait for libsocket to provide a lwls_notice(no_intro  or lwls_notive start / stop
 * insert a few line of code that are quite useless

`
    int i, v = 0, size = argc - 1;

    char *str = (char *)xmalloc(v);

    for(i = 1; i <= size; i++) {
        str = (char *)xrealloc(str, (v + strlen(argv[i])));
        strcat(str, argv[i]);
        strcat(str, " ");
    }

    printf("%s\n", str);
`